src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -308,19 +307,19 @@ namespace PlantLoopEquip {
 		//Pipe has no special types at the moment, so find it this way
 		if ( GeneralEquipType == GenEquipTypes_Pipe ) {
 			if ( EquipTypeNum == TypeOf_Pipe ) {
```

Comment:
Again, when all the components are transitioned into `PlantComponent`s, yes, this will fall down into a single line, eliminating the PlantLoopEquip huge blocks.  But for now, I planned on changing the lines 1-by-1 so that the other devs could see what I was doing easily.  I'd like to leave this one as-is for now.